### PR TITLE
Add Packagist to README.md

### LINKED ISSUE #44: Add Packagist badges to this repo
Please add similar badges to this repo as were added in our [sendgrid-php repo](https://github.com/sendgrid/sendgrid-php/pull/505). Make sure to modify them for the smtpapi-php repo.

### PR DESCRIPTION
**Pull request by edxz**
fix #44

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/smtpapi-php?pullRequest=47) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/smtpapi-php?pullRequest=47) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/smtpapi-php?pullRequest=47) it.</sub>

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ![SendGrid Logo](https://uiux.s3.amazonaws.com/2016-logos/email-logo%402x.png)
 [![BuildStatus](https://api.travis-ci.org/sendgrid/smtpapi-php.png?branch=master)](https://travis-ci.org/sendgrid/smtpapi-php)
 [![Latest Stable Version](https://poser.pugx.org/sendgrid/smtpapi/version.png)](https://packagist.org/packages/sendgrid/smtpapi)
+[![Latest Version on Packagist](https://img.shields.io/packagist/v/sendgrid/sendgrid.svg?style=flat-square)](https://packagist.org/packages/sendgrid/sendgrid)
```
Comment (mbernier) [inline README.md]: Can you please link to https://packagist.org/packages/sendgrid/smtpapi rather than https://packagist.org/packages/sendgrid/sendgrid ?

Comment (edxz) [reply README.md]: sorry, my bad
done

**Comment by mbernier** [comment]
Thank you so much!!

**Comment by thinkingserious** [comment]
Hello @edxz,<br /><br/>Thanks again for the PR!<br />                                              <br />It's <a href="https://sendgrid.com/blog/announcing-hacktoberfest-2017">HACKTOBERFEST</a>! We want to show our appreciation by sending                                              you some special Hacktoberfest swag. If you have not already, could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**